recipe/meta.yaml

```diff
@@ -1,32 +1,29 @@
 {% set name = "sshtunnel" %}
-{% set version = "0.1.3" %}
-{% set sha256 = "598805bb106d8b6d8033a04f2a6b4a64fc9d08a937fe994c7700d0c32d617050" %}
+{% set version = "0.1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ version }}.zip
-  url: https://github.com/pahaz/sshtunnel/archive/{{ version }}.zip
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f29ae41a1bd3afa64e9a31029bece2966e4be9a9641e8262372741e691c40d76
 
 build:
   noarch: python
   entry_points:
     - sshtunnel = sshtunnel:_cli_main
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
-
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed --verbose
 
 requirements:
   host:
-    - pip
     - python
-    - setuptools
+    - pip >=18.1
+    - setuptools >=34.0.0
+    - paramiko >=1.15.2
   run:
-    - python
-    - paramiko
+    - {{ pin_compatible('paramiko') }}
```

Comment:
Why is this needed? And not just paramiko?

Reply:
I implemented pin_compatible to resolve overdepending.  The conda build logs indicate in future versions overdepending will be changed from a warning to an error.

[Pinning at the recipe level](https://github.com/conda/conda-build/blob/master/docs/source/resources/variants.rst#pinning-at-the-recipe-level) has a good description and examples including using lower_bound and upper_bound vs min_pin and max_pin.

Reply:
I've only ever seen this being used for packages with non-Python code (C for example), because the compiler used for those packages should be the same for the packages downstream (if they need to be compiled too.)

AFAIK paramiko is pure Python.

Reply:
pin_compatible removed.